RavenM.Updater/Plugin.cs

```diff
@@ -17,14 +17,33 @@ namespace RavenM.Updater
     class Plugin : BaseUnityPlugin
     {
         static readonly string PLUGIN_DIR = "BepInEx/plugins/";
+        static readonly string GITHUB_TOKEN = "Bearer TOKEN";
```

Comment:
I'm a little nervous about this... personal access tokens shouldn't be revealed even if they have restricted permissions? Someone could spam and rate-limit it, etc.

What do you think about adding https://nightly.link/ to the repo?

Reply:
👍

Reply:
https://nightly.link/iliadsh/RavenM/workflows/main/master/RavenM.zip

Reply:
thanks